packages/jest-get-type/src/index.js

```diff
@@ -19,14 +19,15 @@ export type ValueType =
   | 'regexp'
   | 'map'
   | 'set'
+  | 'date'
   | 'string'
   | 'symbol'
   | 'undefined';
 
 // get the type of a value with handling the edge cases like `typeof []`
 // and `typeof null`
 const getType = (value: any): ValueType => {
-  if (typeof value === 'undefined') {
+  if (value === undefined) {
```

Comment:
`undefined` is read only in ES5 and up, so no reason for this check (unless I'm missing something really obvious)